libaugrim/src/time.rs

```diff
@@ -13,10 +13,11 @@
 // limitations under the License.
 
 use std::cmp::PartialOrd;
+use std::fmt::Debug;
 use std::ops::Add;
 use std::time::Duration;
 
-pub trait Time: Add<Duration, Output = Self> + PartialOrd + Copy {}
+pub trait Time: Add<Duration, Output = Self> + PartialOrd + Copy + Debug {}
 
```

Comment:
Shouldn’t we just but Debug as a constraint on the generics where it’s required?

Reply:
I don't think so because a) its fundamental; and b) seems unfortunate to have to change the API later if you want to log or print something.